Zilon.Core/Zilon.Core/MapGenerators/PrimitiveStyle/SquareMapFactory.cs

```diff
@@ -46,7 +53,16 @@ public static async Task<ISectorMap> CreateAsync(int mapSize)
         {
             var factory = new SquareMapFactory();
 
-            return await factory.CreateAsync(mapSize).ConfigureAwait(false);
+            var squaregenerationOptionsSubScheme = new SquareGenerationOptionsSubScheme { Size = mapSize };
+            var generationOptions = new SectorMapFactoryOptions(squaregenerationOptionsSubScheme);
+
+            return await factory.CreateAsync(generationOptions).ConfigureAwait(false);
+        }
+
+        private class SquareGenerationOptionsSubScheme : ISectorSquareMapFactoryOptionsSubScheme
+        {
+            public SchemeSectorMapGenerator MapGenerator { get; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'MapGenerator'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)